ktxspec.adoc

```diff
@@ -1148,11 +1147,18 @@ have a sample with a channelType that indicates it is alpha.
 ===== DFD for Supercompressed Data
 When `<<_supercompressionscheme,supercompressionScheme>>` is not 0
 the _dfDescriptorBlock_ must preserve the `colorModel`, `transferFunction`,
-`colorPrimaries`, `flags` and `texelBlockDimension[0-3]` of the
-pre-deflation images along with each sample's `channelType`,
-`qualifiers`, `bitlength`, `bitOffset`, `sampleLower` and `sampleUpper`.
-`bytesPlane[0-7]` must be set to 0 to indicate an _unsized format_,
-as described in Section 5.19 _Unsized Formats_ of <<KDF13>>.
+`colorPrimaries`, `flags`, `texelBlockDimension[0-3]` and
+`bytesPlane[0-7]` of the pre-deflation images along with each
```

Comment:
Consider adding a note here reiterating that BasisLZ supercompression works only on ETC1S blocks and therefore implicitly requires specific values for dimensions and `bytesPlane*`.

Reply:
Addressed in https://github.com/KhronosGroup/KTX-Specification/pull/221/commits/4837e331ba415f9acbe5a90c210a40caaca0ac90. Please review.